test/tests/functional/pbs_hook_set_jobenv.py

```diff
@@ -180,32 +182,40 @@ def common_log_match(self, daemon):
                   "TEST_SQUOTE5=music \\'makes \\\"the\\'\\\"people",
                   "TEST_SQUOTE6=loving\\'",
                   "TEST_SPECIAL={}[]()~@#$%^&*!",
-                  "TEST_SPECIAL2=<dumb-test_text>"]
+                  "TEST_SPECIAL2=<dumb-test_text>",
+                  "TEST_RETURN=\\'3\,",
+                  "4\,",
+                  "5\\',"]
```

Comment:
Can you use \n instead of making them their own strings? At the moment, the test checks every line by itself. If `4,` shows up before `TEST_RETURN=...`, the test will pass.

Reply:
Cannot add '\n' here because '\n' is not included in the items of the list returned by log_lines(), (though lines are split by '\n' ). But I have added checks to make sure that the 4 matched is in the line after TEST_RETURN=...